core/examples/http_bench_json_ops.js

```
@@ -41,17 +47,12 @@ function close(rid) {
 }
 
 async function serve(rid) {
-  while (true) {
-    const nread = await read(rid, requestBuf);
-    if (nread <= 0) {
-      break;
+  try {
+    while (true) {
+      await read(rid, requestBuf);
+      await write(rid, responseBuf);
     }
-
-    const nwritten = await write(rid, responseBuf);
-    if (nwritten < 0) {
-      break;
-    }
-  }
+  } catch { /**/ }
```

Print out error instead of swallowing